Content.Shared/Utility/ContentHelpers.cs

```diff
@@ -98,5 +98,42 @@ public static int RoundToNearestLevels(double actual, double max, int levels)
 
             return (int) Math.Round(actual / max * levels, MidpointRounding.AwayFromZero);
         }
+
+        /// <summary>
+        /// Basically helper for when you need to choose 0..N-1 element based on what
+        /// percentage does actual/max takes.
+        /// Example:
+        /// We have a stack of 30 <paramref name="max"/> elements.
+        /// When <paramref name="actual"/> is: 
+        /// - 0..9 we return 0.
+        /// - 10..19 we return 1.
+        /// - 20..30 we return 2.
+        ///
+        /// Useful when selecting N sprites for display in stacks, etc.
+        /// </summary>
+        /// <param name="actual">How many out of max elements are there</param>
+        /// <param name="max"></param>
+        /// <param name="levels"></param>
+        /// <returns>The </returns>
+        /// <exception cref="ArgumentException">if level is one or less</exception>
+        public static int RoundToEqualLevels(double actual, double max, int levels)
+        {
+            if (levels <= 1)
+            {
+                throw new ArgumentException("Levels must be greater than 1.", nameof(levels));
```

Comment:
Will this throw if the cable RSI only has one layer?

Reply:
Yes. A stack of icons with one layer, is just an icon.

Reply:
Something for a different PR then, but it should probably be made to still work even if someone replaced the RSI with only a single layer.